neutron/templates/ml2/ml2_vmware.yaml

```diff
@@ -0,0 +1,8 @@
+
+{{- $context := . -}}
```

Comment:
This is not really needed. The inital scope of "." can be accessed in a range loop by using "$." instead.